### ISSUE TITLE
garbage(?) in `times` on nginx 1.13.12

### ISSUE THREAD

**Issue by discordianfish** [vozlt/nginx-module-vts#121]
Hi,

after upgrading to the nginx ingress controller 0.14.0 which is based on nginx 1.13.12 with vts 0.1.15 (Details: https://github.com/kubernetes/ingress-nginx/blob/master/images/nginx/build.sh), the values in `times` which were unix timestamps before are now some garbage(?) numbers.

Here is the global server zone for example:

```
   "serverZones": {
    "_": {
      "requestCounter": 79,
      "inBytes": 9693,
      "outBytes": 88821,
      "responses": {
          ...
      },
      "requestMsec": 0,
      "requestMsecs": {
        "times": [
          338922835,
          338926107,
          338926192,
          ...
```
Same for a upstream zone:
```
    "default-oauth2-proxy-argo-80": [
      {
        "server": "10.244.116.27:9999",
        "requestCounter": 26,
        "inBytes": 15986,
        "outBytes": 1696178,
        "responses": {
          "1xx": 0,
          "2xx": 9,
          "3xx": 16,
          "4xx": 0,
          "5xx": 1
        },
        "requestMsec": 97,
        "requestMsecs": {
          "times": [
            0,
            0,
            ...,
            339608917,
            339609047,
            339609253
          ],
```

When downgrading to ingress controller 0.11.0, based on nginx 1.13.9 the `times` array contain as expected unix timestamps, e.g:

```
      "requestMsecs": {
        "times": [
          0,
          0,
          0,
          1525191037755,
          1525191040875,
          1525191041141,
          1525191047770,
```

**Comment by discordianfish** [comment]
Looks like `nowMsec` is also affected by this and shows a number in the same ball park as the `times` values. So maybe the data type was changed and it gets cut-off or something? Trying to dig something up in the nginx change log/history.

**Comment by discordianfish** [comment]
Maybe this broke it? https://github.com/nginx/nginx/commit/c7e8a6f2123c653b63ed8013a805eddff502b9ee

This could be related too: https://github.com/nginx/nginx/commit/2d9db482aa92194a2258334545908d620b6dd214

**Comment by vozlt** [comment]
Thanks for the reporting.
I'll fix it soon.

**Comment by vozlt** [comment]
I fixed it and tagged [v0.1.16](https://github.com/vozlt/nginx-module-vts/tree/v0.1.16).
Please test it.
Thanks.

Latest commit: https://github.com/vozlt/nginx-module-vts/commit/2e2c19b230fd24aa86b9f577758a39667ca247f8
